tests/unit/acl-v2.tcl

```diff
@@ -405,6 +405,31 @@ start_server {tags {"acl external:skip"}} {
         assert_equal "OK" [r ACL DRYRUN command-test TYPE rw]
         assert_equal "This user has no permissions to access the 'nothing' key" [r ACL DRYRUN command-test TYPE nothing]
     }
+    
+    test {Test sort commands require some type of permission to execute} {
+    
+        r ACL SETUSER sort-user (%R~v* ~mylist +sort +sort_ro)
+        
+        # we should fail since no full key access
```

Comment:
maybe we should note somewhere in the code (not only on GH comments) that this is a compromise, not a real intent.
i.e. if we some day break / fix that, and this test will fail, it would be nice to understand that it was used to verify the compromises we made in the code working correctly, and the test can be deleted / modified.

such a note should be placed in the code that makes that check too (in sortCommand IMHO)